etna/lib/etna/logger.rb

```diff
@@ -19,6 +18,8 @@ def log_error(e)
       e.backtrace.each do |trace|
         error(trace)
       end
+
+      Rollbar.error(e)
```

Comment:
What does this do if Rollbar isn't configured with a key? Like for local development, or staging or something?

Reply:
It's a simple no op
https://github.com/rollbar/rollbar-gem/blob/master/lib/rollbar/logger_proxy.rb#L25